wiki/Storyboards/zh.md

```diff
@@ -1,21 +1,19 @@
----
-outdated: true
----
+<!--TODO: 抱歉，对故事板理解不够。待补充哈-->
 
-# Storyboards
+# 故事板
 
-![这种场景只有在你游戏进行时得到“完美的300分”时才会出现。)](img/Sbpassing.png "这种场景只有在你游戏进行时得到“完美的300分”时才会出现。")
-![这种场景在你没有击打到前一个组合节拍或者只得到50分时出现。](img/Sbfailing.png "这种场景在你没有击打到前一个组合节拍或者只得到50分时出现。")
+![](img/Sbpassing.png "这种场景只有在你游戏时得到“完美的300分”才会出现。")
+![](img/Sbfailing.png "这种场景在你失误或者只得到50分时出现。")
 
-“故事板”是一系列附加在谱面中的640x480像素的动画背景，一般是为了装饰效果，或者有时候是为了指导游戏进行。故事板可以又图片，动画，声音等多部分组成，不过一般情况下都是以图片为主。主要是为了是谱面看起来更华丽，更吸引玩家的目光。故事板的灵感来源于 osu! 的原版游戏——押忍！战斗!应援团。
+“故事板”（Storyboard,sb）是附加在谱面中的一系列动画背景，一般情况下起到装饰作用，有时候也用于指导游戏进行。故事板可以由图片，动画，声音等多部分组成，不过一般情况下都是以图片为主，主要是为了使谱面看起来更华丽，更吸引玩家。故事板的灵感来源于游戏[押忍！战斗！应援团](/wiki/Glossary#ouendan)。故事板还能根据玩家的表现呈现不同的内容。
 
-故事板一般来说制作很花时间，并且要有一定的动画制作技巧或者图片制作技巧。
+故事板通常存放于谱面文件夹中单独一份[`.osb` 文件](/wiki/osu!_File_Formats/Osb_(file_format))，或储存于[`.osu` 文件](/wiki/osu!_File_Formats/Osu_(file_format))内的`[Event]`条目下。因此，可以在[谱面集](/wiki/Beatmap)中为不同难度创建不同的故事板。
 
```

Comment:
missing a paragraph, for ref:
`Storyboards are often very difficult to make and require considerable time and expertise in standard animation and in the creation of graphics. osu! offers a design editor inside of the beatmap editor to aid the creation of storyboards, but most avid storyboard producers opt to instead program via storyboard scripting directly. Many creators choose to write programs in full-featured programming languages to generate storyboards scripts, because complex visual effects can require a great amount of storyboard code to produce.`